### ISSUE TITLE
'eleventy' command not found / 'eleventy' is not recognized (SOLVED)

### ISSUE THREAD

**Issue by leoossa** [11ty/eleventy#1923]
**Describe the bug**
While trying to use https://github.com/google/eleventy-high-performance-blog I was not able to run any 'eleventy' command though it was installed (not recognized by the shell).

**To Reproduce**
I tried to use:

in 'package.json/package.json.lock' directory:
npm install 
npm install -g 
sudo npm install -g 
npm install --save-dev 
npm install --save-dev -g eleventy 
sudo npm install --save-dev -g eleventy 
npm install @11ty/eleventy 

eventually, that helped:
sudo npm install @11ty/eleventy -g 

**Expected behavior**
eleventy is properly recognized by the shell

**Environment:**

- OS and Version: Manjaro Linux with fish shell
- Eleventy Version @11ty/eleventy@0.12.1

**Comment by pdehaan** [comment]
Yeah, [**eleventy**](https://www.npmjs.com/package/eleventy) is a completely different package than the scoped [**@11ty/eleventy**](https://www.npmjs.com/package/@11ty/eleventy), which can be a bit confusing.

Personally I like to install **@11ty/eleventy** as a devDependency and then use npm scripts to access it (versus installing it globally, or using npx). Which looks similar to how the [**eleventy-high-performance-blog**](https://github.com/google/eleventy-high-performance-blog#6-build-and-test) suggests ([<kbd>npm run build</kbd>](https://github.com/google/eleventy-high-performance-blog/blob/c8773073c2cf62118a2e07a86f2a92d36544eeb4/package.json#L7)). When installing **@11ty/eleventy** (<kbd>npm i @11ty/eleventy -D</kbd>) it will add an `eleventy` executable (symlink, really) to your ./node_modules/.bin/ folder:
```sh
lrwxr-xr-x    eleventy -> ../@11ty/eleventy/cmd.js
```
But when you run an npm script via <kbd>npm run build</kbd>, it automatically adds "./node_modules/.bin/" to your `$PATH` variable. If you wanted to run **@11ty/eleventy** without using npm scripts [for reasons], I guess you could try <kbd>$(npm bin)/eleventy --version # 0.12.1</kbd>.
Some people also use `npx eleventy`, but I don't really like that solution since you can get very odd behavior depending on whether you've run <kbd>npm install @11ty/eleventy</kbd> locally or not. If you have **@11ty/eleventy** installed locally, it will run that. If you don't have your node modules installed locally it will fetch the incorrect "eleventy" (not the scoped "@11ty/eleventy") package and nothing will work.
I usually use global packages as an absolute last resort, because I switch node versions frequently, and I find global packages confusing and you aren't always certain which versions of global packages it's using and I think local modules are just easier to manage.

**Comment by zachleat** [comment]
We do have a pretty hefty warning about this on https://www.11ty.dev/docs/usage/

Is your issue solved here?

**Comment by leoossa** [comment]
@zachleat @pdehaan actually you're both right. That's a bit my fault (taking a peek at the `package.json` I saw `eleventy` command so I thought the package name should be `eleventy` as well). On the other hand, the funny thing is that's one of the problems package managers were meant to solve. Ambiguity with this package naming is quite a mess. 

It was solved at the time I posted it 
> eventually, that helped:
sudo npm install @11ty/eleventy -g

but I just wanted to share it in case other people will stumble upon the same problem
